### PR TITLE
Add new AppIcons for iOS and Android

### PR DESCRIPTION
**Pull request by LBeul**
Implement a fitting icon for both the iOS and Android Launcher instead of the generic Flutter Logo. The Icon was built by myself in Figma and consists of some usual geometric shapes, so no copyright issues with that.

ball

### CODE REVIEW

**Comment by scitbiz** [comment]
@LBeul Great, thanks! Could you please make a transparent background version for android? Cause it looks better if someone like to theming their OS (like using an icon pack with a custom icon background, rounded icon, etc).

And also don't include `pubspec.lock` (because your pull request only add icons, not changing the code) and generated files (such as ` .flutter-plugins-dependencies`, ` ios/Flutter/flutter_export_environment.sh`). I forget to ignore them in .gitignore, will add next time. Sorry!

I'm currently refactoring the code and it would be nice if this icon can be used as the refresh indicator and loading indicator icon. If you could add a transparent background version icon (`pokeball-logo.png` - 40x40) to the `assets/images` folder, it will be awesome.

Thanks again!

**Comment by LBeul** [comment]
Hey there.

In commit _26bb53a_, I just swapped the Icons for new ones with transparent background and added both a png and svg to the assets as requested by @scitbiz :)

File: .flutter-plugins-dependencies [hunk 1]
```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"path_provider","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\path_provider-1.4.0\\\\","dependencies":[]},{"name":"sqflite","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\sqflite-1.1.7+2\\\\","dependencies":[]}],"android":[{"name":"path_provider","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\path_provider-1.4.0\\\\","dependencies":[]},{"name":"sqflite","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\sqflite-1.1.7+2\\\\","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"path_provider","dependencies":[]},{"name":"sqflite","dependencies":[]}],"date_created":"2020-06-13 14:19:33.076805","version":"1.17.2"}
```
Comment (scitbiz) [inline .flutter-plugins-dependencies:1]: Please uncommit this file

File: android/settings_aar.gradle [hunk 2]
```diff
@@ -0,0 +1 @@
+include ':app'
```
Comment (scitbiz) [inline android/settings_aar.gradle:1]: Please uncommit this file

File: ios/Flutter/flutter_export_environment.sh [hunk 3]
```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
```
Comment (scitbiz) [inline ios/Flutter/flutter_export_environment.sh:1]: Please uncommit this file

File: pubspec.lock [hunk 4]
```diff
@@ -7,28 +7,28 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.10"
```
Comment (scitbiz) [inline pubspec.lock:10]: Please uncommit this file

**Comment by scitbiz** [comment]
@LBeul Please rebase your code

**Comment by LBeul** [comment]
@scitbiz should work now

**Comment by scitbiz** [comment]
@LBeul You pull request still have conflicts, please rebase and squash into one commit only

**Comment by scitbiz** [comment]
I'll close this issue here because there is no action coming up and the master branch has changed. If you still want to add this, please open a new pull request based on the new master branch. Thank you!

**Pull request closed without merging**